best-practices/README.md

```diff
@@ -0,0 +1,71 @@
+Best Practices
+=============
+
+The following are locally agreed upon best practices. As often as possible,
+these will inherit from practices within the larger open source development
+community at large. These best practices may, and likely will, change in the
+future. But that change will be incorporated via group consensus and documented
+here.
+
+* [Documentation](documentation.md)
+* [Project README's](project_readme.md)
+* [Deployment](deployment.md)
+
+
+General
+-------
+
+* These are not to be blindly followed; strive to understand these and ask
+  when in doubt.
+* Don't duplicate the functionality of a built-in library.
+* Don't swallow exceptions or "fail silently."
```

Comment:
I think this is generally good advice, but I've definitely seen times where we really didn't care about an exception we were handling.  Maybe this could be reworded to something like "In general, exceptions should be logged or passed up the stack; clearly documents the rare cases where you want to completely ignore an exception"?